docs/content/components/localtime.md

```diff
@@ -0,0 +1,64 @@
+---
+title: LocalTime
+status: Alpha
+source: https://github.com/primer/view_components/tree/main/app/components/primer/local_time.rb
+storybook: https://primer.style/view-components/stories/?path=/story/primer-local-time-component
+---
+
+import Example from '../../src/@primer/gatsby-theme-doctocat/components/example'
+import RequiresJSFlash from '../../src/@primer/gatsby-theme-doctocat/components/requires-js-flash'
+
+<RequiresJSFlash />
+
+<!-- Warning: AUTO-GENERATED file, do not edit. Add code comments to your Ruby instead <3 -->
+
+Add a general description of component here
+Add additional usage considerations or best practices that may aid the user to use the component correctly.
+
+## Accessibility
+
+Add any accessibility considerations
```

Comment:
I feel like this shouldn't be here if there are no `@accessibility` YARD comments.